src/Sylius/Behat/Resources/config/suites/api/channel/channels.yml

```diff
@@ -0,0 +1,20 @@
+# This file is part of the Sylius package.
+# (c) Paweł Jędrzejewski
+
+default:
+    suites:
+        ui_channels:
+            contexts:
+                - sylius.behat.context.hook.doctrine_orm
+
+                - sylius.behat.context.transform.channel
+
+                - sylius.behat.context.setup.channel
+
+                - sylius.behat.context.api.channel
+
+                - sylius.behat.context.api.shop.homepage
+
+                - sylius.behat.context.setup.product
```

Comment:
From what I can see, the convention of order is `hook` -> `transform` -> `setup` -> `api/ui/etc`.
Also, why is it spaced so much?